.github/workflows/ci-gh1.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         SCCACHE_REGION: us-east-2
         SCCACHE_BUCKET: rapids-sccache-devs
         SCCACHE_S3_KEY_PREFIX: legate-cunumeric-dev
-        VAULT_HOST: ${{ secrets.PERSONAL_ACCESS_TOKEN && 'https://vault.ops.k8s.rapids.ai' || '' }
+        VAULT_HOST: ${{ secrets.PERSONAL_ACCESS_TOKEN && 'https://vault.ops.k8s.rapids.ai' || '' }}
```

Comment:
Any ideas why the job started failing with this change? (and why it was passing before this)